code/modules/emotes/definitions/audible.dm

```diff
@@ -78,10 +79,6 @@
 	emote_message_3p = "USER snores."
 	conscious = 0
 
-/decl/emote/audible/whimper
```

Comment:
Was this removed? Can't see it readded anywhere.

Reply:
It was a duplicate definition for the emote.